ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -872,3 +874,331 @@ def test_ipa_migrate_with_log_file_option(self):
+        uid/gid for user on remote and local IPA server.
+        """
+        username = 'produser1'
+        run_migrate(
+            self.replicas[0],
+            "prod-mode",
+            self.master.hostname,
+            "cn=Directory Manager",
+            self.master.config.admin_password,
+            extra_args=['-n']
+        )
+        result1 = self.master.run_command(['id', username])
+        result2 = self.replicas[0].run_command(['id', username])
+        assert result1.stdout_text == result2.stdout_text
+
+    def test_ipa_migrate_subids(self, empty_log_file):
+        """
+        This testcase checks that subids for users are migrated
+        to the local server from the remote server
+        """
+        base_dn = str(self.master.domain.basedn)
+        user_name = 'admin'
+        DEBUG_MSG = (
+            "Added entry: cn=subids,cn=accounts,{}"
+        ).format(base_dn)
+        CMD_MSG = (
+            "1 subordinate id matched\n"
+            "Owner: admin\n"
+        )
+        run_migrate(
+            self.replicas[0],
+            "prod-mode",
+            self.master.hostname,
+            "cn=Directory Manager",
+            self.master.config.admin_password,
+            extra_args=['-n']
+        )
+        cmd = self.replicas[0].run_command(
+            ['ipa', 'subid-find',
+             '--owner', user_name]
+        )
+        install_msg = self.replicas[0].get_file_contents(
+            paths.IPA_MIGRATE_LOG, encoding="utf-8"
+        )
+        assert DEBUG_MSG in install_msg
+        assert CMD_MSG in cmd.stdout_text
+
+    def test_ipa_migrate_check_service_status(self):
+        """
+        This testscase checks that ipactl and sssd
+        services are running post ipa-migrate tool
+        successful runs completed
+        """
+        run_migrate(
+            self.replicas[0],
+            "prod-mode",
+            self.master.hostname,
+            "cn=Directory Manager",
+            self.master.config.admin_password,
+            extra_args=['-n']
+        )
+        cmd1 = self.replicas[0].run_command([
+            "ipactl", "status"
+        ])
+        assert cmd1.returncode == 0
+        cmd2 = self.replicas[0].run_command([
+            "systemctl", "status", "sssd"
+        ])
+        assert cmd2.returncode == 0
+
+    def test_custom_idrange_is_migrated(self, empty_log_file):
+        """
+        This testcase checks that custom idrange is migrated
+        from remote server to local server in production
+        mode.
+        """
+        range_name = "testrange"
+        CMD_OUTPUT = (
+            "1 range matched\n"
+            "Range name: testrange \n"
+            "First Posix ID of the range: 10000\n"
+            "Number of IDs in the range: 10000\n"
+            "First RID of the corresponding RID range: 300000\n"
+            "First RID of the secondary RID range: 400000\n"
+            "Range type: local domain range\n"
+        )
```

Comment:
The string compare will fail because you are missing the dashes in the output of the ipa command.